pkg/controller/operator/operator_controller.go

```diff
@@ -117,6 +117,13 @@ func (r *ReconcileOperator) Reconcile(request reconcile.Request) (reconcile.Resu
 		return reconcile.Result{}, err
 	}
 
+	// License must be accepted
+	if !instance.Spec.LicenseSpec.Accept {
+		licenseErr := errors.NewBadRequest("License must be accepted in order to proceed.")
+		return reconcile.Result{}, licenseErr
+		klog.Error("License was not accepted.", licenseErr)
```

Comment:
1. log error before return, otherwise, it won't be printed. 
2. I don't think we want to return this as an error coz it triggers another reconcile. let's update it in the status. There fore we need to enhance the status little bit.  in addition to just a PodStatus, we need something like
{
  Phase string
  Reason *string
  Message *string
  PodStatus *PodStatus  
}